IM_calculation/IM/structural_response.py

```diff
@@ -330,7 +335,7 @@ def calculate_structural_response(
     storey_moment = np.cumsum(moment[::-1], axis=0)[::-1]
 
     # calculates total acceleration (floor acceleration) matrix
-    ground_accel = np.tile(g * acc_time_history, (storey + 1, 1))
+    ground_accel = np.tile((1 + g * 8.81) * acc_time_history, (storey + 1, 1))
```

Comment:
That seems a bit magicy for me. Can't work out the best way to organise this part though.

Would it be easiest/best it convert the acc_time_history at the start of the function so it's "consistent" all the way through?